src/hooks/useActivityData.ts

```
@@ -0,0 +1,182 @@
+        Collection.SESSIONS,
+        sessionId,
+        SessionsSubcollection.ATTENDEES
+      );
+
+      // Query all attendees - we'll filter by role
+      const attendeesQuery = query(attendeesRef);
+      
+      // Query for each role type
+      const [campers, staff, admins] = await Promise.all([
+        executeQuery<CamperAttendeeID, CamperAttendee>(
+          attendeesQuery as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)